modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/GlobalSnapshotStateChannelEventsProcessor.scala

```diff
@@ -71,7 +71,9 @@ object GlobalSnapshotStateChannelEventsProcessor {
           case hash if hash == Hash.empty => SnapshotFeesInfo.empty.pure // genesis
           case _ =>
             deserialize[Signed[CurrencyIncrementalSnapshot]](event.snapshotBinary).flatMap {
-              case None => new Exception("Could not get snapshot after deserializing").raiseError[F, SnapshotFeesInfo]
+              case None =>
+                logger.warn(s"Could not get snapshot fee info after deserializing event $event, using empty snapshot fees");
+                SnapshotFeesInfo.empty.pure
```

Comment:
I can't speak to if this is acceptable behavior but wouldn't you need `logger.warn(...) >> SnapshotFeesInfo.empty.pure[F]` to have the log output to console?

Reply:
:facepalm: that got lost in a reformatting